MANIFEST.in

```diff
@@ -1,3 +1,5 @@
+# https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html
```

Comment:
Add a comment something along the lines of:

# For more information on managing files in pacakge

Reply:
The links now have a description.